.github/workflows/pull_request.yml

```diff
@@ -23,10 +27,14 @@ jobs:
           node-version: lts/*
       - name: Install dependencies
         run: npm ci
+      - name: Ensure docker is present
+        run: docker --version
       - name: Install Playwright Browsers
         run: npx playwright install --with-deps
       - name: Run Playwright tests
-        run: npx playwright test
+        run: |
+          docker pull amazon/dynamodb-local:latest
```

Comment:
Why do you need to pull it first? Is this because of the timeout?

If so, then this is a potential failure mode for local development too, isn't it?

Reply:
The image is generally not present, but I could probably make the plugin smarter and check for the image and pull it if it isn't present, I'll play around with that

Reply:
The Docker CLI normally pulls images if they are not available. Does the API not do that?

Reply:
Nope, without this, there is an error that there is no such image

Reply:
OK, then please update your plugin to pull the image first.